app/src/forms/submission/controller.js

```diff
@@ -16,6 +17,15 @@ module.exports = {
     } catch (error) {
       next(error);
     }
+  },
+  email: async (req, res, next) => {
+    try {
+      const submission = await service.read(req.params.formSubmissionId, req.currentUser);
+      const response = await emailService.submissionConfirmation(submission.form.id, req.params.formSubmissionId, req.body, req.headers.referer);
```

Comment:
Typo - referrer, not referer. Definitely need to make sure we're not forcing API calls to use the wrong spelling.